packages/mdc-slider/index.js

```diff
@@ -114,14 +132,23 @@ class MDCSlider extends MDCComponent {
         setTrackFillStyleProperty: (propertyName, value) => {
           this.trackFill_.style.setProperty(propertyName, value);
         },
+        focusThumb: () => {
+          this.thumb_.focus();
+        },
+        activateRipple: () => {
```

Comment:
`MDCRipple` takes care of registering handlers that activate/deactivate the ripple, so it's not necessary to do this in the slider and we can remove `activateRipple()`/`deactivateRipple()` from the adapter.

Reply:
These are also used to activate the ripple when the user clicks on anything but the thumb. If the user was to click on the track and drag the ripple would not be activated and would appear like a bug, so i manually activate the ripple when the user does not click on the thumb directly.